azure/v11.4.0/README.md

```diff
@@ -19,6 +19,10 @@ Along with azure-operator and NGINX IC, minor improvements were also made to Cor
 
 **Note for SEs:**  
 
+Before cluster upgrade to 11.4.0, check if the DNS zone has a TXT DNS record called `<clusterid>ingress.<clusterid>.<region>.azure.gigantic.io`.
+If the record is not there, but there is another TXT one called `ingress.<clusterid>.<region>.azure.gigantic.io`, please copy the latter using the right name (with the cluster id at the beginning).
```

Comment:
copy it to where? 

(not sure if this could be clarified, or just my lack of knowledge)

Reply:
let me clarify it

Reply:
better now @cokiengchiara ?